spec/factories/instance_users.rb

```diff
@@ -1,7 +1,12 @@
 FactoryGirl.define do
   factory :instance_user do
     instance
-    user
+    user do
+      User.skip_callback(:create, :after, :create_instance_users)
```

Comment:
this looks nasty and not thread-safe.

anyway to create a user without an instance?